apimonitor/__init__.py

```
@@ -0,0 +1,7 @@
+from __future__ import annotations
+
+# Initialize Celery app when Django starts
+from .celery import app as celery_app  # noqa: F401
```

Ruff (0.12.2)

Unused `noqa` directive (unused: `F401`)

Remove unused `noqa` directive

(RUF100)